ksql-engine/src/main/java/io/confluent/ksql/planner/LogicalPlanner.java

```diff
@@ -76,6 +77,10 @@ public LogicalPlanner(
   public OutputNode buildPlan() {
     PlanNode currentNode = buildSourceNode();
 
+    if (!analysis.getTableFunctions().isEmpty()) {
+      currentNode = buildFlatMapNode(currentNode, functionRegistry);
```

Comment:
nit: no need to pass field to non-static method.

Reply:
done